src/js/views/globals/app-nav/app-nav_views.js

```diff
@@ -118,7 +184,7 @@ const AppNavCollectionView = CollectionView.extend({
 
 const PatientsAppNav = View.extend({
   template: hbs`
-    <h3 class="app-nav__search app-nav__link js-search">{{far "magnifying-glass"}}{{ @intl.globals.appNav.appNavViews.patientsAppNav.searchTitle }}</h3>
+    <h3 class="flex app-nav__search js-search">{{fas "magnifying-glass"}}{{ @intl.globals.appNav.appNavViews.patientsAppNav.searchTitle }}</h3>
```

Comment:
Is the far version still in use?  If not we should remove it from the package.json

Reply:
It's used in two different places. The patient search modal and `Find In List...` inputs.